File: src/com/yijiwenhua/backend/model/SysRole.java
package com.yijiwenhua.backend.model;

import java.io.Serializable;

/**
 * 系统角色
 */
public class SysRole implements Serializable {
	private static final long serialVersionUID = 1L;

	/**
	 * 角色有效
	 */
	public static final int ROLE_ENABLED = 1;

	/**
	 * 角色无效
	 */
	public static final int ROLE_UN_ENABLED = 0;

	/** role_id */
	private Long roleId;

	/** 上级角色ID,0为顶级 */
	private Long parentId;

	/** 角色名称 */
	private String roleName;

	/** 描述 */
	private String descn;

	/** 是否生效(0:无效,1:有效) */
	private Integer enabled;

	/** role_id */
	public Long getRoleId() {
		return roleId;
	}

	public void setRoleId(Long roleId) {
		this.roleId = roleId;
	}

	/** 上级角色ID,0为顶级 */
	public Long getParentId() {
		return parentId;
	}

	public void setParentId(Long parentId) {
		this.parentId = parentId;
	}

	/** 角色名称 */
	public String getRoleName() {
		return roleName;
	}

	public void setRoleName(String roleName) {
		this.roleName = roleName;
	}

	/** 描述 */
	public String getDescn() {
		return descn;
	}

	public void setDescn(String descn) {
		this.descn = descn;
	}

	/** 是否生效(0:无效,1:有效) */
	public Integer getEnabled() {
		return enabled;
	}

	public void setEnabled(Integer enabled) {
		this.enabled = enabled;
	}

}

File: WebRoot/static/validatejs/SysRole.js
var saveSysRoleConfig = {
	parentId:[{rule:integer,msg:"上级角色ID,0为顶级 必须是数字"}],
	roleName:[{rule:notBlank,msg:"角色名称 不能为空"},{rule:length,params:{min:0,max:50},msg:"角色名称 长度不在0-50范围"}],
	descn:[{rule:length,params:{min:0,max:500},msg:"描述 长度不在0-500范围"}],
	enabled:[{rule:integer,msg:"是否生效(0:无效,1:有效) 必须是数字"}]
};
